core/ir/x86/instr.c

```diff
@@ -50,10 +73,11 @@
 void
 instr_set_x86_mode(instr_t *instr, bool x86)
```

Comment:
Perhaps we want to change all uses of instr_set_x86_mode() to use instr_set_isa_mode() instead?

Reply:
Hmm I see this in the docs:
```
api/docs/release.dox: - Deprecated instr_set_x86_mode(), replacing it with instr_set_isa_mode().
```

Yet we still have internal uses?  Did we only deprecate external calls?

Reply:
Oh, I'd suppose so.
Internally they are still in a few places (e.g., https://github.com/DynamoRIO/dynamorio/blob/96c132d84290d6164990a41581f228d405008275/core/arch/emit_utils_shared.c#L2338 and https://github.com/DynamoRIO/dynamorio/blob/96c132d84290d6164990a41581f228d405008275/core/arch/mangle_shared.c#L1895).
If we decide to remove them, should that work go in a separate PR?

Reply:
Yes, but seems lower priority.